capstone/src/main/java/com/google/sps/data/Form.java

```diff
@@ -26,8 +26,4 @@ public String getEditURL() {
     public String getURL() {
         return (String) super.entity.getProperty(Form.URL_PROPERTY_KEY);
     }
-
-    public Entity getFormEntity() {
-        return super.entity;
-    }
```

Comment:
I think we can have this here? Except there shouldn't be a time we down cast a lesson into a form so this actually may not be needed.

Reply:
we have a getEntity() in Lesson.java that should work for all the classes that extend it

Reply:
I could be wrong though

Reply:
Yeah that makes sense. It would not work in the case of Form form = new Form(Blah) but for our case, we would not need to do that in our servlets.

Reply:
NB it would actually work fine. The lesson entity getter returns the exact same (super.) entity that this method is, so whether you created it as a form or just got it as a lesson that happens to be a form, they are editing the same underlying form